RInAppMessaging/Classes/IAMPreferenceBuilder.swift

```diff
@@ -25,12 +25,30 @@
         return self
     }
 
+    @objc @discardableResult
+    public func setIDTrackingIdentifier(_ idTrackingIdentifier: String) -> IAMPreferenceBuilder {
```

Comment:
@imaginaris  I'd expect all our public API to be documented. Please create a tech debt record entry for that.